from-attribute.js

```diff
@@ -24,7 +32,12 @@ function initializeFromAttribute (propertyName, ctr, attributeName) {
 	if (ctr[metaSymbol]._attributeChangedCallbackHandler === undefined) {
 		ctr[metaSymbol]._attributeChangedCallbackHandler = {};
 	}
-	if (attributeName === undefined) {
+	if (typeof attributeName === 'object') {
+		if (isJSONLike(attributeName)) {
```

Comment:
Can we give a warning when the wrong thing is passed? This can happen in a future PR.